app/assets/javascripts/megatron/utils/text-helpers.js

```diff
@@ -50,7 +50,7 @@ var TextHelpers = {
     }
     Array.prototype.forEach.call(document.querySelectorAll('textarea:not(.no-auto-size)'), autoHeight)
 
-    bean.on(document.querySelector('body'), 'keyup', 'textarea', function(event){
+    bean.on(document.querySelector('body'), 'keyup, toggler:show', 'textarea', function(event){
```

Comment:
Good question. Looks like it actually shouldn't have the comma:

Example:
`bean.on(element, 'keydown keyup', handler);`

See [https://www.npmjs.com/package/bean](https://www.npmjs.com/package/bean)

It's working, though, at least for the toggler event. The keyup event may be broken with the errant comma.